src/_assets/css/_bootstrap_adjust.scss

```diff
@@ -19,3 +19,21 @@ a.card {
     }
   }
 }
+
+.btn {
+  font-family: $site-font-family-alt;
+}
+
+code {
+  background-color: $flutter-color-light-blue;
```

Comment:
Why use blue? More often on other sites (like developers and www), I've see gray used, such as #f7f7f7. /cc @Sfshaza